driver-redpanda/src/main/java/io/openmessaging/benchmark/driver/redpanda/RedpandaBenchmarkDriver.java

```diff
@@ -103,7 +104,7 @@ public void initialize(File configurationFile, StatsLogger statsLogger) throws I
                     deletes.all().get();
                 }
             } catch (ExecutionException e) {
-                if (e.getCause() instanceof UnknownTopicOrPartitionException) {
+                if (e.getCause() instanceof UnknownTopicOrPartitionException || e.getCause() instanceof UnknownTopicIdException) {
```

Comment:
I could not actually determine the precise difference between these two exceptions, but anyway Kafka client throws this during the deletion race when the cluster is Kafka.